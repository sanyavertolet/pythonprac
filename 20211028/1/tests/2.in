print(', '.join([str(i) for i in fib(5, 10)]))
