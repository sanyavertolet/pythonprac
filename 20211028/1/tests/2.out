8, 13, 21, 34, 55
